__tests__/schema/taxonomy-term/delete-entity-links.ts

```diff
@@ -46,7 +47,7 @@ beforeEach(() => {
         ...taxonomyTermCurriculumTopic,
         childrenIds: [],
       })
-      return res(ctx.json({ success: true }))
+      return HttpResponse.json({ success: true })
```

Comment:
```suggestion
      return Response.json({ success: true })
```

Should also work in our case (I guess...)